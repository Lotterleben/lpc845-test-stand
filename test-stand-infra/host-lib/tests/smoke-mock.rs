use host_lib::{
    mock::Mock,
    assistant::Assistant,
    conn::Conn,
};

#[test]
fn make_instance() {
    let mock = Mock::new();
    let test_hdl = mock.clone();

    let conn = Conn::from_serial_port(Box::new(mock)).unwrap();
    let assistant = Assistant::new(conn);

    assert!(test_hdl.is_totally_empty());

}
