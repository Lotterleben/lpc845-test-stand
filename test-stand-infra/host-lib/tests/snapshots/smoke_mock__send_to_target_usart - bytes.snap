---
source: tests/smoke-mock.rs
expression: "&msg"

---
[
    1,
    1,
    6,
    4,
    254,
    237,
    218,
    122,
    0,
]
